lib/utils.js

```diff
@@ -35,4 +35,42 @@ function isStream (stream) {
   return stream !== null && typeof stream === 'object' && typeof stream.pipe === 'function'
 }
 
-module.exports = { isGzip, isDeflate, isStream }
+// It is a helper used to provide a async iteratable for
+// Readable.from
+async function * intoAsyncIterator (payload) {
```

Comment:
I though we should just clone `into-stream` including `License` inside this repository.
Because the logic here is very similar to `into-stream@6` but a bit simpler.

Reply:
cc @mcollina

Reply:
Imho the logic is trivial. So no licensing needed.